django/db/models/fields/related.py

```diff
@@ -1314,7 +1314,7 @@ class ForeignObjectRel(object):
 
     # Reverse relations are always nullable (Django can't enforce that a
     # foreign key on the related model points to this model).
-    null = True
+    null = False
```

Comment:
The diff doesn't make it obvious that two classes are affected. This is for `ForeignObjectRel`while the below is for `ManyToManyField`. The second assertion in the new `model_fields` test fails like this on 1.8:
```
    self.assertFalse(AllFieldsModel._meta.get_field('reverse2').null)
AttributeError: 'ManyToOneRel' object has no attribute 'null'
```